js-functions/js-functions_functions-with-parameters-extra/js/index.js

```diff
@@ -18,9 +20,13 @@ printSquare(5);
 */
 
 // Uncomment the following function calls and implement the function `printCircumference`.
+function printCircumference(number) {
+  const circumference = 2 * 3.14159 * number;
```

Comment:
You can actually use `Math.PI` for pi.

```suggestion
  const circumference = 2 * Math.PI * radius;
```